tools/miner/miner.go

```diff
@@ -52,7 +52,7 @@ func (eg *eventGenerator) GenerateMinerEvents(ctx context.Context, transactions
 
 		actorAddress := tx.TxTo
 		// this is executed by(from) the miner actor
-		if tx.TxType == parser.MethodUpdateClaimedPower {
+		if tx.TxType == parser.MethodUpdateClaimedPower || tx.TxType == parser.TotalFeeOp {
```

Comment:
What happens if the tx_fee op is not there anymore, and it is included on the main tx row? how can we cover that?